src/v/datalake/translation/partition_translator.cc

```diff
@@ -275,7 +275,8 @@ partition_translator::do_translate_once(retry_chain_node& parent_rcn) {
           read_begin_offset,
           read_end_offset,
           _partition->last_stable_offset());
-        _partition->probe().update_iceberg_translation_offset_lag(0);
+        _partition->probe().update_iceberg_translation_offset_lag(
```

Comment:
the metric calculates offset lag, if we are caught up, lag = 0? (note: here we are updating the probe directly).

Reply:
Ah oops you're right. Something does seem off in Brandon's testing 🤦